examples/doc-examples/access-token.tf

```diff
@@ -0,0 +1,8 @@
+provider "pingfederate" {
+  access_token                        = "accesstokenvaluefromclient"
+  https_host                          = "https://localhost:9999"
+  admin_api_path                      = "/pf-admin-api/v1"
+  insecure_trust_all_tls              = true
```

Comment:
These new example files are great. It might be good to remove the `insecure_trust_all_tls` and `x_bypass_external_validation_header` from these since they're more for development and not relevant to authentication.